limiter/limiter.go

```diff
@@ -24,11 +25,15 @@ func New(maxRequests, rps int, maxDuration time.Duration) *Run {
 	}
 
 	r := &Run{
+		unlimited:    *atomic.NewBool(maxRequests == 0),
```

Comment:
We access `unlimited` from `Stop` and from `More`. `More` is running on the goroutines making requests, while `Stop` will run from either a timer goroutine, or the main goroutine waiting for user requests to end. So we'll have two accesses to the same variable from different goroutines, where one is a write, so it's a data race.

We can actually test this by making it a non-atomic bool, and running the tests with `-race`, and we get:
```
Read at 0x00c4200161e0 by goroutine 24:
  github.com/yarpc/yab/limiter.(*Run).More()
      /Users/prashant/gocode/src/github.com/yarpc/yab/limiter/limiter.go:45 +0x3f
  github.com/yarpc/yab/limiter.TestUnlimitedRequests()
      /Users/prashant/gocode/src/github.com/yarpc/yab/limiter/limiter_test.go:80 +0x1ce
  testing.tRunner()
      /usr/local/Cellar/go/1.7/libexec/src/testing/testing.go:610 +0xc9

Previous write at 0x00c4200161e0 by goroutine 25:
  github.com/yarpc/yab/limiter.(*Run).Stop()
      /Users/prashant/gocode/src/github.com/yarpc/yab/limiter/limiter.go:63 +0x8b
  github.com/yarpc/yab/limiter.(*Run).Stop-fm()
      /Users/prashant/gocode/src/github.com/yarpc/yab/limiter/limiter.go:35 +0x41
```